repl/src/test/scala/com/cloudera/livy/repl/Python3InterpreterSpec.scala

```diff
@@ -208,15 +208,5 @@ class Python3InterpreterSpec extends BaseInterpreterSpec {
     ))
   }
 
-  // Scalastyle is treating unicode escape as non ascii characters. Turn off the check.
-  // scalastyle:off non.ascii.character.disallowed
-  it should "print unicode correctly" in withInterpreter { intp =>
-    intp.execute("print(u\"\u263A\")") should equal(Interpreter.ExecuteSuccess(
```

Comment:
because, that case not working python3.
```
    intp.execute("""print("\xE2\x98\xBA")""") should equal(Interpreter.ExecuteSuccess(
      TEXT_PLAIN -> "\u263A"
    ))
```
return \xE2\x98\xBA not \u263A
i try fix this. but need more time.
could you help if you do not mind